cmd/root/root.go

```diff
@@ -114,10 +114,15 @@ func Execute(ctx context.Context, cmd *cobra.Command) error {
 		if err == nil {
 			logger.Info("completed execution",
 				slog.String("exit_code", "0"))
+		} else if errors.Is(err, ErrAlreadyPrinted) {
+			logger.Debug("failed execution",
+				slog.String("exit_code", "1"),
+			)
```

Comment:
Why not also at info?

Reply:
My thinking was that since error is already printed we do not need to print it again. So I've been deciding between not printing it at all or printing at debug level.